internal/errorlint/errorlint.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2023 Outreach Corporation. All Rights Reserved.
+
+// Description: See package comment for this one file package.
+
+// Package errorlint contains the necessary logic for the error/log/trace linter.
+package errorlint
```

Comment:
Two things here from me would be:
1. Ideally this package comment is a little more descriptive as to what this linter actually provides (see https://github.com/getoutreach/lintroller/blob/main/internal/why/why.go#L5-L7)
2. (nit, just my personal opinion) I think if I was you I'd just drop the `lint` postfix on `errorlint` - we already assume its a linter.

Reply:
I took inspiration from nolint, should have took it from why , ill change it

Reply:
error is interface so probably keep lint with error or you can suggest some other name